src/engine/guidance/post_processing.cpp

```diff
@@ -298,6 +303,10 @@ std::vector<RouteStep> postProcess(std::vector<RouteStep> steps)
 
 void trimShortSegments(std::vector<RouteStep> &steps, LegGeometry &geometry)
 {
+#if OSRM_POST_PROCESSING_PRINT_DEBUG
```

Comment:
The problem here is, that we need to merge some stuff while it is augmented further.
When I remove this, I have to add it for all further improvements and remove it again.